components/CustomerInfo/CustomerInfo.js

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from "react";
+import useSWR from "swr";
+
+import ReactModal from "react-modal";
+import {
+  CustomerInfoInput,
+  InfoText,
+  InfoContainer,
+  CustomerInfoForm,
+  CustomerInfoButtonContainer,
+} from "./styles";
+
+import { GreenButton, RedButton } from "../Buttons/styles";
+import { Form } from "../../pagestyles/styles";
+
+export default function CustomerInfo({ customer, id }) {
+  const [isExpanded, setIsExpanded] = useState(false);
+  const [isModalOpen, setIsModalOpen] = useState(false);
+
+  const { mutate } = useSWR("/api");
```

Comment:
The mutate function only updates your view when you specify the exact same api endpoint as you use for the PATCH request. Currently, your page does not update the change. It only updates when you click outside the browser window and back in. Instead, what you want is to immediately react to the database change.

```suggestion
  const { mutate } = useSWR(`/api/${id}`);
```

Reply:
Thanks! I was looking for the reason, why the customerinfo isnt displayed before refreshing